apps/utils/scripts/run-bfd-pipeline

```diff
@@ -153,6 +167,26 @@ while getopts "b:c:d:i:Il:r:R:s:t:xzZh" option ; do
   x) install=false ;;
   z) debug_enabled=true ;;
   Z) debug_enabled=true ; debug_suspend=y ;;
+  -) echo $OPTARG
```

Comment:
Ignorant question, is this `echo` leftover from testing?

Reply:
Indeed it is!  Thanks for catching that.